simpledotorg/simple-server
# Add details section to diabetes reports

**Story card:** 
[sc-7785](https://app.shortcut.com/simpledotorg/story/7785/add-diabetes-coverage)
[sc-7787](https://app.shortcut.com/simpledotorg/story/7787/add-diabetes-patient-registrations-and-follow-ups-card)
[sc-7786](https://app.shortcut.com/simpledotorg/story/7786/add-lost-to-follow-up-card-under-details)

## Because

We need to report the following data
- Diabetes coverage
- Lost to follow up patients
- Diabetes Patient Registrations and Follow-ups

## This addresses

- Updated the estimated population to be unique to a `region` and `diagnosis`
- Updated `facility_group` to edit the diabetes population of the region
- Updated the repository and added views to render the graphs and the table 
- Add necessary definitions in the footnote

## Test instructions

- Turn on diabetes_management_reports on the flipper
- Visit the reports page of a facility which has enable_diabetes_management set to true
- Click on the Diabetes link
- You should see the  
  - Diabetes patient registrations and follow-ups table in the `Compare Performance` section
  - Diabetes Coverage graph in the `Details` section
  - Lost to follow-up graph  in the `Details` section

## Screenshots:

<img width="1352" alt="Screenshot 2022-05-24 at 1 19 37 PM" src="https://user-images.githubusercontent.com/32141642/169978112-df1a9829-3a58-4966-942d-32f2df353c97.png">

<img width="1352" alt="Screenshot 2022-05-24 at 1 19 22 PM" src="https://user-images.githubusercontent.com/32141642/169978098-0fc4523f-394e-4775-b827-3fd7cbf63fd7.png">
